scripts/street_sign_recognizer.py

```diff
@@ -17,21 +17,40 @@ def __init__(self):
         """ Initialize the street sign reocgnizer """
```

Comment:
Nice job, looks like a good project overall. There are a couple of little things I mentioned later, but generally looks like you got the idea.